### ISSUE TITLE
Rename qt6 kirigami2 packages to kirigami.

### ISSUE THREAD

**Issue by micwoj92** [Martchus/PKGBUILDs#187]
Relevant packages:
mingw-w64-kirigami2
android-x86-kirigami2
android-x86-64-kirigami2
android-armv7a-eabi-kirigami2
android-aarch64-kirigami2

Your modifications to the build environment (e.g. changes in `makepkg.conf`): N/A

Question/request or brief description of the problem:
Name of this package changed to just kirigami, old qt5 version is/was kirigami2. You can see this in packages extra/kirigami2 and extra/kirigami as well as download links:

https://download.kde.org/stable/frameworks/6.2/ Has kirigami

https://download.kde.org/stable/frameworks/5.116/ Has kirigami2

**Comment by Martchus** [comment]
Maybe I'll rename it when updating the package anyway (even though there is no advantage in needlessly changing names of packages).

**Comment by Martchus** [comment]
I guess I'll also have to get rid of other variants besides aarch64 on the AUR or upload the required Qt packages. Not sure whether it is worth it. Probably only aarch64 is actually useful anyway at this point.

I am not sure anymore but I guess for actually deploying an APK it was also required to have some additional KDE packages. Those might need to be uploaded to the AUR as well. Again, this is only useful if anyone is actually interested in those packages.

So are you actually interested in the package (or just want to clean up the AUR)? If not, maybe it is the best to delete everything on the AUR after all and I just use the PKGBUILDs from my own repo where I develop them as far as I need them without annoyng streamlining efforts.

**Comment by micwoj92** [comment]
> Maybe I'll rename it when updating the package anyway (even though there is no advantage in needlessly changing names of packages).

Thanks. There would be advantage. kf5  software should depend on kirigami2 where kf6 kirigami. Obviously this is more "what if" situation here, as nothing depends on these. This is just for correctness with upstream naming that for some reason changed.


> I guess I'll also have to get rid of other variants besides aarch64 on the AUR or upload the required Qt packages. Not sure whether it is worth it. Probably only aarch64 is actually useful anyway at this point.
>
> I am not sure anymore but I guess for actually deploying an APK it was also required to have some additional KDE packages. Those might need to be uploaded to the AUR as well. Again, this is only useful if anyone is actually interested in those packages.
> 
> So are you actually interested in the package (or just want to clean up the AUR)? If not, maybe it is the best to delete everything on the AUR after all and I just use the PKGBUILDs from my own repo where I develop them as far as I need them without annoyng streamlining efforts.

If these packages are updated then I have no problem, but it seems like you don't do automatic pushes from this repo to AUR, so it seems like they are abandoned when they are not. I'm not interested in these packages (well, maybe mingw variant as I sometimes do crosscompile to test things on wine/windows, but that also is "what if").
So yes, I'm just filling requests to clean up AUR.

IMO no need to delete (everything) if these packages would actually get pushes regularly from this repo using some automation.

**Comment by Martchus** [comment]
All kirigami packages for archs other then aarch64 have been cleaned up for now. That's likely for the best because so far I only build/test aarch64 and I needed to provide Qt 6 first. I re-uploaded the remaining packages (android-aarch64-kirigami and mingw-w64-kirigami) under a new name without the "2" and filed merge requests for the old packages. At this point there's no use in keeping the old kf5 packages around anymore (and the Android one has already been updated to be the kf6 version anyway). With that I'm closing this issue.

> IMO no need to delete (everything) if these packages would actually get pushes regularly from this repo using some automation.

Note that I have some scripts I usually invoke so the pushing is semi-automatic. Sometimes I needed to add completely new packages to the AUR (like in this case) which creates some extra effort, though.
